automatic/umlet/README.md

```diff
@@ -0,0 +1,3 @@
+Free UML Tool for Fast UML Diagrams.
```

Comment:
Readme has a standard form. Yoiu miss screenshot, features, link to package etc.